### ISSUE TITLE
Comment parsing fails if other `@` comments are included

### ISSUE THREAD

**Issue by Tam** [kristiandupont/kanel#622]
I'm using [Postgraphile](https://postgraphile.org) to generate a GraphQL schema from a Postgres database. Postgraphile has the concept of [Smart Tags](https://postgraphile.org/postgraphile/next/smart-tags) that, similar to the `@type` comment, use the `@` symbol to be recognised.

If I have a comment like this, everything works fine:
```sql
comment on column public.attachment.file is E'@type(FileMetadata, ''../../services/Types.d'', true, false, true)';
```
```ts
file: FileMetadata;
```

But if I start including Postgraphile Smart Tags, it breaks:
```sql
comment on column public.attachment.file is E'
@behaviour -*
@type(FileMetadata, ''../../services/Types.d'', true, false, true)
';
```
```ts
/** \n@behaviour -*\n@type(FileMetadata, '../../services/Types.d', true, false, true)\n */
file: unknown;
```

Incidentally, Postgraphile also has a [`@type`](https://postgraphile.org/postgraphile/next/polymorphism) comment. So far I haven't run into conflicts with it, but that's probably because I haven't used either very much!

**Comment by kristiandupont** [comment]
Right, that would be a problem! I guess I should make those tags optional as I don't think many people use them and it's probably better solved with a hook instead. I will ponder this.
